### ISSUE TITLE
The number in this cell is formatted as text or preceded by an apostrophe

### ISSUE THREAD

**Issue by narainsagar** [PHPOffice/PHPExcel#722]
Hi, While exporting html to xlsx,
 All of the cells with numbers show an error message next to them on Excel. When I read the error message it says that "they are formatted as text or preceded by an apostrophe". 

Here is screenshot attached!
![661590224-error](https://cloud.githubusercontent.com/assets/13800762/11002260/7f8fad46-84cd-11e5-811d-0298f8f8e752.png)

It just looks a bit ugly, that's all..

Why is that occurs?, How to fix this? 

What i did was quite simple here is my sample code:

if(!is_dir( TMP_FILES )) {
	mkdir( TMP_FILES, 0777 );
}
//TMP_FILES == '../temp/';
$htmlfile = TMP_FILES . $filename . '.html';
file_put_contents($htmlfile, $content);

$objReader = new PHPExcel_Reader_HTML;
$objPHPExcel = $objReader->load($htmlfile); // load .html file

// Set properties
$objPHPExcel->getProperties()->setCreator("Hello World");
$objPHPExcel->getProperties()->setLastModifiedBy("Hello World");
$objPHPExcel->getProperties()->setTitle("Office 2007 XLSX Document Report");
$objPHPExcel->getProperties()->setSubject("XLSX Report");
$objPHPExcel->getProperties()->setDescription("XLSX report document for Office 2007");

$excelFile = TMP_FILES . $filename . '.xlsx'; // excel file to generate.
// Creates a writer to output the $objPHPExcel's content
$objWriter = PHPExcel_IOFactory::createWriter($objPHPExcel, 'Excel2007');
$objWriter->save($excelFile);
unlink($htmlfile); // remove .html file

Though everything  else works fine.

Thanks!

**Comment by StefanGous** [comment]
Have you managed to find a solution to this? I am facing the same problem.

**Comment by narainsagar** [comment]
Yeah, I just removed the "Pound" sign and it works fine.. Though if we remove the space between "Pound" and "number" it also works fine... Just make sure that the cell value should be string(text). That's it.
